### ISSUE TITLE
Update Wagtail and Django versions for fec-cms repo

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fec-cms#1538]
We are [currently](https://github.com/18F/fec-cms/blob/develop/requirements.txt) using Wagtail version 1.8 and Django 1.10. We should upgrade to the next major or at least the next minor version.

"Extended support" for these versions is ending or has already ended.
https://www.djangoproject.com/download/#supported-versions
http://docs.wagtail.io/en/v1.13.1/releases/upgrading.html

From @ccostino:
Django 2.0 is ~going to be coming out soon~ out now. There's definitely an argument for sticking with 1.11 due to the fact that it's the current LTS release, but Django releases between LTS versions tend to be extremely stable and don't occur all that often (every 6 months).

Need to also watch out for wagtailadmin. It's currently being used to make sure the most recent version of jquery is used rather than the wagtail default.

- [ ] Upgrade to LTS (1.11) version of Django and latest version of Wagtail
- [ ] Testing will be completed in a separate ticket

**Comment by patphongs** [comment]
Quick update:

Need to attempt to rewrite the following custom editor functions in wagtail 2.0 using the new draft.js framework:

- [ ] Blockquote
- [ ] Sans seriff
- [ ] Edit HTML

Circle tests are also failing, need to investigate and fix before I can deploy to feature for testing.
- [ ] Investigate failing circle tests
- [ ] deploy to feature for content owner testing

**Comment by patphongs** [comment]
**Update!** 
Most of this work is complete. I will separate out the rest of the issues below into additional smaller issues:

- [ ] Add Edit HTML button into the editor (https://github.com/fecgov/fec-cms/issues/1971)
- [ ] Rework django python tests to pass so that we can deploy upgrade (https://github.com/fecgov/fec-cms/issues/1972)
- [ ] Heavily test wagtail 2.0 to make sure the functionality will be adequate for launch (https://github.com/fecgov/fec-cms/issues/1973)

**Comment by patphongs** [comment]
The bulk of this work is complete. A few smaller issues came out of upgrading Wagtail that needs to be addressed before we can fully deploy the upgrade to production.

- [ ] Add Edit HTML button into the editor (https://github.com/fecgov/fec-cms/issues/1971)
- [ ] Rework django python tests to pass so that we can deploy upgrade (https://github.com/fecgov/fec-cms/issues/1972)
- [ ] Heavily test wagtail 2.0 to make sure the functionality will be adequate for launch (https://github.com/fecgov/fec-cms/issues/1973)
